src/DMS/Service/Meetup/Response/SelfParsingResponse.php

```diff
@@ -44,29 +30,21 @@ public function getData()
     }
 
     /**
-     * @param array $metadata
-     */
-    public function setMetadata($metadata)
-    {
-        $this->metadata = $metadata;
-    }
-
-    /**
-     * @return array
+     * Makes Meetup API Data available on the Data attribute
      */
-    public function getMetadata()
```

Comment:
where did metadata go? we still need to keep that, again, its a break in BC and a fallback due to Meetup.com weird behaviour...